apis/api-media/src/schema/cloudflare/r2/asset.spec.ts

```
@@ -1,5 +1,7 @@
 import { graphql } from 'gql.tada'
 
+import { Video } from '.prisma/api-media-client'
```

ESLint

[error] Useless path segments for ".prisma/api-media-client", should be "./.prisma/api-media-client"

(import/no-useless-path-segments)